### ISSUE TITLE
Evolbuntu vivid dark volume icon issues

### ISSUE THREAD

**Issue by cvetan** [franksouza183/EvolvereSuit#110]
There appears to be some issues with icons for dark panels. For instance volume icon in panel remains all the black, so it makes it look bad and hard to look. I have used only evolbuntu, so i can't tell for other themes.

**Comment by franksouza183** [comment]
Have you tried the dark-fallback / dark-light-fallback variants?

**Comment by cvetan** [comment]
Yes, i did. In fact the problem that i described is in dark fallback version. Since i am using it with ambiance theme.

**Comment by franksouza183** [comment]
Could you post a screenshot?

**Comment by cvetan** [comment]
http://www.dodaj.rs/f/s/13o/4bm7U2eI/evolvere-vivid-dark.png

http://www.dodaj.rs/f/33/vt/3zbJRzYe/evolbuntu-dark-fallback.png

I've posted links to pictures. Check out it in full size so you can see what i am talking about. :)

**Comment by franksouza183** [comment]
Please, test now

**Comment by cvetan** [comment]
OK, only i have to clone repository again. I will report back. :)

Also, why doesn't noobslab update ppa? There are no evolbuntu-vivid variants at all. And the ppa is last updated 5 weeks ago. I know this is not your job, but i was wondering if you have insight in that.

**Comment by franksouza183** [comment]
I believe that only for Evolvere final releases

**Comment by cvetan** [comment]
No, the volume icon is still all black. It's the same. :( 
Just tried out, all dark-fallback versions have error.
